scene/resources/environment.cpp

```diff
@@ -287,6 +288,12 @@ void Environment::_validate_property(PropertyInfo &property) const {
 		}
 	}
 
+	if (property.name == "tonemap_white") {
+		if (tone_mapper == TONE_MAPPER_LINEAR) {
+			property.usage = PROPERTY_USAGE_NOEDITOR | PROPERTY_USAGE_INTERNAL;
+		}
+	}
```

Comment:
```suggestion
	if (property.name == "tonemap_white" && tone_mapper == TONE_MAPPER_LINEAR) {
		property.usage = PROPERTY_USAGE_NOEDITOR | PROPERTY_USAGE_INTERNAL;
	}
```
Can these be merged into one if statement? They are in your master version PR.

Reply:
I followed the existing code style in each branch, so I don't think it's important to change here.

Reply:
Oh ok, I didn't know about the code style difference.